addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComLighting4Message.java

```diff
@@ -74,43 +80,56 @@ public static SubType fromByte(int input) throws RFXComUnsupportedValueException
     }
 
     public enum Commands {
-        UNDEFINED_0(0),
-        ON(1),
-        UNDEFINED_2(2),
-        UNDEFINED_3(3),
-        OFF(4);
+        OFF_0(0, false),
+        ON_1(1, true),
+        OFF_2(2, false),
+        ON_3(3, true),
+        OFF_4(4, false),
+        ON_5(5, true),
+        ON_7(7, true),
+        ON_9(9, true),
+        ON_12(12, true),
+        UNKNOWN(-1, false);
```

Comment:
I like the idea of putting them in the XML, but then we are duplicating the info in the class. Is there a way we can avoid that?

Also, how does the user configure this? It seems more complicated than other devices, so should probably be documented somewhere.